CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 ## 3.74.0 (Unreleased)
 
+FEATURES:
+* **New Resource:** `google_app_engine_service_network_settings` ([#3371](https://github.com/hashicorp/terraform-provider-google-beta/pull/3371))
+* **New Resource:** `google_vertex_ai_dataset` ([#3369](https://github.com/hashicorp/terraform-provider-google-beta/pull/3369))
+* **New Resource:** google_cloudbuild_worker_pool ([#3372](https://github.com/hashicorp/terraform-provider-google-beta/pull/3372))
+
+IMPROVEMENTS:
+* bigtable: added `cluster.kms_key_name` field to `google_bigtable_instance` ([#3354](https://github.com/hashicorp/terraform-provider-google-beta/pull/3354))
+* composer: added field `max_pods_per_node` to resource `google_composer_environment` (beta) ([#3376](https://github.com/hashicorp/terraform-provider-google-beta/pull/3376))
+* secretmanager: added `ttl`, `expire_time`, `topics` and `rotation` fields to `google_secret_manager_secret` ([#3360](https://github.com/hashicorp/terraform-provider-google-beta/pull/3360))
+
+BUG FIXES:
+* Exported fields in Mapper struct. ([#3375](https://github.com/hashicorp/terraform-provider-google-beta/pull/3375))
```

Comment:
is this for a specific service? is it really a bug? or can it be none?

Reply:
This should be none.